src/Domain/Structure/Neighbors.cpp

```diff
@@ -9,10 +9,32 @@
 #include <pup_stl.h>
 
 #include "Domain/Structure/ElementId.hpp"
+#include "Utilities/Algorithm.hpp"
 #include "Utilities/ErrorHandling/Assert.hpp"
 #include "Utilities/GenerateInstantiations.hpp"
 #include "Utilities/StdHelpers.hpp"
 
+namespace {
+template <size_t VolumeDim>
+bool ids_and_orientations_are_consistent(
+    const std::unordered_set<size_t>& block_ids,
+    const std::unordered_map<size_t, OrientationMap<VolumeDim>>& orientations) {
+  return alg::none_of(block_ids, [&orientations](size_t block_id) {
```

Comment:
Ranges..??? :( :(

Reply:
can't use ranges until we get rid of gcc9

Reply:
💔